engine/src/main/java/org/terasology/engine/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -279,8 +279,25 @@ private boolean initializeOpenVRLibrary() {
         if (initialized) {
             return true;
         }
-        logger.info("Adding OpenVR search path: " + NativeHelper.getOpenVRLibPath());
-        NativeLibrary.addSearchPath("openvr_api", NativeHelper.getOpenVRLibPath());
+
+        String target = OS.USER_DIRECTORY +  OS.PATH_SEPARATOR + "openvr_natives" + OS.PATH_SEPARATOR;
```

Comment:
I'd prefer to use `Paths...` operations here instead of manually constructing the path string. Would that be possible or is module sandboxing preventing that?